### DISCUSSION TITLE
easy installation script

### DISCUSSION THREAD

**Discussion by zning** [tailwindlabs/tailwindcss#4379] [Ideas]
Thanks for this great library! I'm wondering is there any plan to setup a quick npx command to install tailwindcss.
Something like what react or storybooks did.
For storybooks install, all you need to do is to run "npx sb init" and it will auto detect your package and install and setup everything for you! That's super easy to test and adopt.
I'm thinking about something like: "npx tailwindcss init "or alike.
Any suggestions?

**Comment by quick007** [comment]
Use `npx tailwindcss init -p`. Don't think it install tailwind though, so you'll need to to `npm install -D tailwindcss@latest postcss@latest autoprefixer@latest` yourself

**Reply by zning** [reply]
I meant that one npx script will auto detect the package and install and setup everything for you(No craco or css setups). Current npx tailwindcss init will simply add a config.js file in the root folder. I hope it can do everything. 
Not sure how hard to build something like that...
